config/routes.rb

```diff
@@ -113,6 +113,8 @@
     mount Sidekiq::Web, at: 'sidekiq'
   end
 
+  get 'models', to: 'model_graph#index'
+  get 'sha', to: ->(_env) { plain_text_response(ENV.fetch('GIT_REV')) }
```

Comment:
(Mostly unrelated change; just grouping these slightly related/similar routes together and formatting them similarly.)